### DISCUSSION TITLE
Is there a recommended way to override invoice templates?

### DISCUSSION THREAD

**Discussion by ex5** [saleor/saleor#9797] [Q&A] [answered]
As a merchant I'd like to be able to modify invoice templates to make them look more familiar to customers (e.g. current templates are missing tax rate percentages and don't mention "VAT" explicitly, just "Tax") and, possibly, add branding to them.

Is there a recommended way to override invoice templates?

There's always a possibility of modifying the `templates/` directory, however this doesn't seem like a clean production solution, because this would either require a fork or a clone with unstaged changes.

**Comment by jbergstroem** [comment, marked as answer]
It depends a bit on how you deploy it, but if you are in container land you can always just mount your own `templates` folder (`docker run -v mytemplates:/app/templates -it ...`).
